packages/cli/_test.pony

```diff
@@ -38,33 +38,34 @@ class iso _TestMinimal is UnitTest
 
     let cmd = match cmdErr | let c: Command => c else error end
 
+    h.assert_eq[String]("minimal", cmd.fullname())
     h.assert_eq[Bool](true, cmd.option("aflag").bool())
 
 class iso _TestBadName is UnitTest
   fun name(): String => "ponycli/badname"
 
-  fun apply(h: TestHelper) ? =>
+  // Negative test: command names must be alphanum tokens
+  fun apply(h: TestHelper) =>
     try
-        let cs = CommandSpec.leaf("min imal", "")
-    else
-        return // error was expected
+      let cs = CommandSpec.leaf("min imal", "")
+      h.fail("expected error on bad command name" + cs.name())
```

Comment:
You probably want to append `: ` to your string literal here.

Otherwise the `cs.name()` bit is going to get printed squished up against the word "name" from the string literal when printed.